ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master__4.3.yaml

```diff
@@ -38,12 +38,12 @@ tag_specification:
   namespace: ocp
 tests:
 - as: e2e-aws-ocp-43-continuous
-  cron: 0 */12 * * *
+  cron: 0 */12 * * 1-5
```

Comment:
For posterity from crontab: "At minute 0 past every 12th hour on every day-of-week from Monday through Friday."